Modules/Scripted/DICOMLib/DICOMExportScalarVolume.py

```diff
@@ -107,24 +108,24 @@ def export(self):
         """
         cliparameters = {}
         # Patient
-        cliparameters['patientName'] = self.tags['Patient Name']
-        cliparameters['patientID'] = self.tags['Patient ID']
-        cliparameters['patientBirthDate'] = self.tags['Patient Birth Date']
-        cliparameters['patientSex'] = self.tags['Patient Sex'] if self.tags['Patient Sex'] else "[unknown]"
-        cliparameters['patientComments'] = self.tags['Patient Comments']
+        cliparameters['patientName'] = self.tags[_('Patient Name')]
```

Comment:
I don't think these should _not_ be translated. They are set in DICOMScalarVolumePlugin.py, etc. where they are not translated either.

Reply:
Yes, sorry, I meant that these should not be translated. I'm not sure why these keys chosen to be human-readable-looking strings.

Reply:
Thanks @lassoan and @pieper for your feedback. They will be taken into account in the next commits.

Regards.